src/app/components/recent-tags/RecentTags.css

```diff
@@ -9,9 +9,17 @@
   height: 100%;
 }
 
+.recent-tags__fancy-border {
+  padding-bottom: 1rem;
+}
+
 .recent-tags__card {
-  overflow: hidden;
-  padding-bottom: 0;
+  /* nextjs route change result in inconsistent stylesheet load orders */
```

Comment:
I would rather the override be done here than in the fancy border component CSS so that fancy border CSS can remain a contextually agnostic component